README.md

```diff
@@ -0,0 +1,78 @@
+# ReactOS Project
+
+Current version: __0.4.6__
+![ReactOS Logo](https://reactos.org/wiki/images/0/02/ReactOS_logo.png)
+
+## Quick Links
+- [Website](https://reactos.org)
+- [Wiki](https://reactos.org/wiki)
+- [JIRA Bug tracker](https://jira.reactos.org/issues)
+- [ReactOS Git mirror](https://git.reactos.org)
+
+## What is ReactOS?
+
+ReactOS™ is an Open Source effort to develop a quality operating system that is
+compatible with applications and drivers written for the Microsoft® Windows™ NT
+family of operating systems (NT4, 2000, XP, 2003, Vista, Seven).
+
+The ReactOS project, although currently focused on Windows Server 2003
+compatibility, is always keeping an eye toward compatibility with
+Windows Vista and future Windows NT releases.
+
+The code of ReactOS is licensed under [GNU GPL 2.0+](https://spdx.org/licenses/GPL-2.0+.html).
+
+## Building ReactOS
+
+To build the system it is strongly advised to use the _ReactOS Build Environment
+(RosBE)._ Up-to-date versions for Windows and for Unix/GNU-Linux are available
+from our download page at: http://www.reactos.org/wiki/Build_Environment/
+
+Alternatively one can use Microsoft Visual C++ (MSVC) version 2010+, together
+with separate installations of CMake and the Ninja build utility.
+
+#### Binaries
+
+To build ReactOS run `ninja`, or alternatively run
```

Comment:
This seems confusing since it won't work unless you also ran configure/CMake

Reply:
That's correct. You *need* the configure step. You can inspire yourself from https://www.reactos.org/wiki/Building_ReactOS#Prerequisites .